hub/templates/hub/area.html

```diff
@@ -120,9 +120,11 @@ <h3 class="h5"><a href="{% url 'area' area_type=overlap_constituency.old_area.ar
                 </a>
             </li>
           {% endif %}
+            {% if is_westminster_cons %}
```

Comment:
Would be nice if this was two-space indented, like the conditional above ;-)

(I try to always do this inside Django templates, as per the "suggestion" in https://pages.mysociety.org/coding-standards.html)